README.md

```diff
@@ -84,22 +84,28 @@ Array<{
 }>;
 ```
 
-### `/api/txs/txBodies`
+### `/api/getAccountState`
 
 #### Input
 
 ```js
 {
-  txsHashes: Array<string>
+  addresses: Array<string> // hex of reward stake addresses
 }
 ```
 
 #### Output
 
 ```js
 {
-  [key: string]: string
+  [addresses: string]: RewardInfo
 };
+type RewardInfo = null | {|
+      poolOperator: null, // not implemented yet
+      remainingAmount: number, // current remaining awards
```

Comment:
these should all be strings

Reply:
indeed they actually are - the readme denoted them incorrectly.  fixed.